Cargo.toml

```diff
@@ -17,7 +17,6 @@ authors = [
 edition = "2021"
 license = "MIT"
 repository = "https://github.com/extendr/extendr"
-rust-version = "1.70"
```

Comment:
THIS is the hero bit. Need to add to changelog

Reply:
I don't really think so, because extendr 0.6.0 didn't have MSRV 1.70. So it is "back to normal" 😃